packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -444,9 +482,11 @@ export async function rebuildNativeModules(
 export async function compileCompassAssets(
   compassPath = COMPASS_PATH
 ): Promise<void> {
+  process.env.COVERAGE = 'true';
```

Comment:
So basically: When we compile compass here, we get coverage. But not when testing the packaged app. So I guess right now this is only really useful when run locally?

Reply:
I guess that's the only way. Kinda like going back to running e2e tests against main except with coverage? Wondering if I should do it in this PR or another.